tests/foreman/api/test_contentview.py

```diff
@@ -1596,6 +1596,7 @@ def test_negative_non_readonly_user_actions(self):
             entities.ContentView(id=content_view.id).read()
 
 
+@skip_if_bug_open('bugzilla', 1625783)
```

Comment:
Having a query on it ..
As Bugzilla `1625783` is in verified state and afaik related tests would execute if any Bug get verified ?

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1625783

Is in the NEW state.

Reply:
thanks , I was referring the `1696988` .. 
Looks good now :)